tools/themis/src/utils.rs

```diff
@@ -4,8 +4,8 @@ use cargo_metadata::{camino::Utf8PathBuf, CargoOpt, MetadataCommand};
 
 use super::types::{Package, Workspace};
 
-pub fn read_toml(path: &Utf8PathBuf) -> anyhow::Result<toml::Value> {
-    Ok(toml::from_slice(&fs::read(path)?)?)
+pub fn read_toml(path: &Utf8PathBuf) -> anyhow::Result<Option<toml::Value>> {
```

Comment:
Look at the users of this function, I wonder if we should introduce a better return type here.  

I think we care about the following three variants:
- everything succeeded and here is the toml value.
- something failed in trying to read the file.
- something failed in try to parse it into a toml.

I wonder if introducing an explicit type that captures the above three variants might make the code more readable.

Reply:
Constrained the error dismissal to ENOENT, anything else will bubble up to a runtime error. At this point, I believe a `Result<Option<T>>` should suffice.